VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -259,6 +259,22 @@ protected void onUpdate() {
                     isLastRotationOn = isRotationOn;
                 }
             };
+            getView().findViewById(R.id.panel_video_only_on_web).setOnClickListener(new OnClickListener() {
+                @Override
+                public void onClick(View v) {
+
+                    final StringBuffer urlStringBuffer = new StringBuffer();
+                        if (! videoEntry.url.startsWith("http://") && ! videoEntry.url.startsWith("https://")) {
```

Comment:
@aleffert  the logic applied here is the same as the one used in existing  View-in-Web function.
Additional validation is handled in   BrowserUtil.open( activity, url);